src/app/campaign/form/campaign-form.container.ts

```diff
@@ -16,20 +19,26 @@ import { withLatestFrom } from 'rxjs/operators';
   `,
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class CampaignFormContainerComponent {
-  campaign$: Observable<Campaign> = this.campaignStoreService.localCampaign$;
+export class CampaignFormContainerComponent implements OnInit {
+  campaign$: Observable<Campaign>;
   accounts$: Observable<Account[]> = this.accountService.accounts;
   advertisers$: Observable<Advertiser[]> = this.advertiserService.advertisers;
 
   constructor(
     public campaignStoreService: CampaignStoreService,
+    public store: Store<any>,
     private accountService: AccountService,
     private advertiserService: AdvertiserService
   ) {}
 
+  ngOnInit() {
+    this.campaign$ = this.store.pipe(select(selectLocalCampaign));
+  }
+
   campaignUpdateFromForm(newState: { campaign: Campaign; changed: boolean; valid: boolean }) {
     const { campaign, changed, valid } = newState;
     this.campaignStoreService.setCampaign({ localCampaign: campaign, changed, valid });
+    this.store.dispatch(new CampaignFormUpdate({ campaign, changed, valid }));
```

Comment:
still doing both campaign store and NgRx store form updates, but campaign store will be removed eventually